### DISCUSSION TITLE
await supabase.from("test0516").select("*") ;  statusText: "Not Found",

### DISCUSSION THREAD

**Discussion by dong-lufei** [supabase/supabase#6849] [Questions]
const res = await supabase.from("test0516").select("*")
console.log(666, res) // { error: {}, data: null, count: null, status: 404, statusText: "Not Found", body: null }
表是存在的而且有一行记录数据，为什么查询得到不

**Comment by dong-lufei** [comment]
const supabase = createClient(SUPABASE_URL, SUPABASE_ANON_KEY)
这里的SUPABASE_URL 是https://<project_ref>.supabase.co  不是https://<project_ref>.supabase.co/rest/v1/？？？

**Reply by steve-chavez** [reply]
In which schema did you create `test0516`? The API exposes the `public` schema by default.

**Reply by dong-lufei** [reply]
在数据库的表中创建的test0516，应该属于public。我是不是应该在auth schema下创建

**Reply by steve-chavez** [reply]
No, the `auth` schema should never be exposed(it has private user data) through the API. Also, since it's managed by Supabase you shouldn't really create any table/view/function in it.

What's the SUPABASE_URL you're using?

**Reply by dong-lufei** [reply]
https://lhzjtaxbhpwmiebsisye.supabase.co

**Reply by steve-chavez** [reply]
I see, I believe you need to change it to: `https://lhzjtaxbhpwmiebsisye.supabase.co/rest/v1/`(this is the REST API url)

**Reply by dong-lufei** [reply]
我改成rest那个url 查询出来的是status: 404, statusText: "Not Found" 其他代码都没动  去掉“/rest/v1/”就查得出来数据：

`import { createClient } from "https://esm.sh/@supabase/supabase-js"

// const SUPABASE_URL = "https://lhzjtaxbhpwmiebsisye.supabase.co/rest/v1/setting"  // { error: {}, data: null, count: null, status: 404, statusText: "Not Found", body: null }
const SUPABASE_URL = "https://lhzjtaxbhpwmiebsisye.supabase.co"  // status: 200
const SUPABASE_ANON_KEY =  "不变"

const supabase = createClient(SUPABASE_URL, SUPABASE_ANON_KEY)
const selectRes = await supabase.from("setting").select("*")
console.log("selectRes", selectRes)`

**Reply by dong-lufei** [reply]
如何查auth里的user表？
const { data, error, statusText } = await supabase.from("users").select("*") 这查的是public里的user表

**Reply by dong-lufei** [reply]
const res = await supabase.auth.signIn({
  email: "xx@qq.com",
  password: "Pwxxx"
})
console.log(res) // 为什么请求失败：{ data: null, user: null, session: null, error: { message: "Request Failed" } }
